.github/workflows/_check_is_rust.yaml

```diff
@@ -0,0 +1,35 @@
+name: Check if app is based on Rust
+
+on:
+  workflow_call:
+    outputs:
+      is_rust:
+        description: Returns "true" if the app is using Rust SDK, else returns "false"
+        value: ${{ jobs.get-is-rust.outputs.is_rust }}
+
+jobs:
+  get-is-rust:
```

Comment:
Why the sudden `-` when it's `_` everywhere else?

Reply:
it's `-`on _get_workflow_version which I used as a template. But will change.